attachment_unindex_content/i18n/es_AR.po

```diff
@@ -0,0 +1,21 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# 	* attachment_unindex_content
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 16.0+e\n"
```

Comment:
Update only .pot file, the rest will be automatically regenerated by the bots and Weblate